apache/zeppelin
# [ZEPPELIN-1047] Add filter to handle upper case

### What is this PR for?
The notebook filter in navbar does not filter the results of note if note name is given in upper case .


### What type of PR is it?
[Bug Fix]

### Todos

### What is the Jira issue?
[ZEPPELIN-1047](https://issues.apache.org/jira/browse/ZEPPELIN-1047)

### How should this be tested?
1.Open the notebook filter under navbar -> Notebook menu.
2.Search for any note by giving name in upper case.
3.You should be able to search all the note book names irrespective of case. 

### Screenshots (if appropriate)

**BEFORE:**
http://g.recordit.co/jqo7DYXStI.gif

**AFTER:**
http://g.recordit.co/uuCT79uEti.gif



### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
